package.json

```diff
@@ -182,8 +182,8 @@
     "mongoose-v6": "npm:mongoose@^6.12.7",
     "mongoose-v7": "npm:mongoose@^7.6.10",
     "morgan": "^1.10.0",
-    "mssql": "^10.0.3",
-    "mssql-v9": "npm:mssql@^9.0.1",
```

Comment:
Suggestion: Only run v11 tests.
If there is no "real change" between major versions, there is no need to run both versions.

Reply:
I just noticed that the adapted test is anyway wrong. You are just iterating over the versions but you are not using both versions (mock missing).

Reply:
My bad. The logic existed already 👍